python/cudf/cudf/core/column/column.py

```diff
@@ -528,6 +528,11 @@ def __setitem__(self, key: Any, value: Any):
         if out:
             self._mimic_inplace(out, inplace=True)
 
+    def _wrap_binop_normalization(self, other):
+        if other is cudf.NA:
+            return cudf.Scalar(other, dtype=self.dtype)
+        return self.normalize_binop_value(other)
```

Comment:
1. Should this method be internal?
2. The type annotation of the base method seems wrong, it handles scalar and arrays but the type annotation says `ScalarLike`.

Reply:
1. It probably should be internal, but the interfaces aren't very clear right now. I plan to change the code so that `binary_operator` also becomes internal and `Frame` code will start using operators (e.g. `a + b` rather than `a.binary_operator(b)`. When I make that changes I'll start solidifying the API.